src/components/badge/badge.css

```diff
@@ -0,0 +1,26 @@
+
+.avatar-online::after,
+.avatar-ofline::after,
+.avatar-away::after {
+  content: "";
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  width: 3rem;
+  height: 3rem;
+  border-radius: 50%;
+  box-sizing: border-box;
+  border: 4px solid var(--light-bg);
+}
+.avatar-online::after {
+  background: var(--online-color);
+}
+.avatar-ofline::after {
+  background: var(--ofline-color);
+}
+.avatar-away::after {
+  background: var(--secondary-text);
+}
+svg  {
+  color: yellow;
```

Comment:
Writing class/variable can be better instead of element/color name. Rest all looks great.

Reply:
Thank you for the review, i will correct it